### PR TITLE
Add more info for siren

### PR DESCRIPTION
**Pull request by piitaya**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/dialogs/more-info/controls/more-info-siren.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    if (newVal === DEFAULT_TONE_KEY) {
+      delete this._selectedTone;
+      return;
+    }
+
+    this._selectedTone = newVal;
+
+    if (this.stateObj?.state === "on") {
+      this._turnOn();
+    }
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-siren.ts:33]: It is not possible to set a tone without turning it on? So selecting a tone when the siren is off doesn't do anything? It is not saved?

File: src/dialogs/more-info/controls/more-info-siren.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                naturalMenuWidth
+                @selected=${this._toneChanged}
+                @closed=${stopPropagation}
+              >
+                <mwc-list-item value=${DEFAULT_TONE_KEY}>
+                  default
+                </mwc-list-item>
+                ${stateObj.attributes.available_tones!.map(
+                  (tone) => html`
+                    <mwc-list-item .value=${tone}>${tone}</mwc-list-item>
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-siren.ts:81]: Use `ha-list-item`

File: src/dialogs/more-info/controls/more-info-siren.ts (hunk 1 above)
Comment (piitaya) [reply src/dialogs/more-info/controls/more-info-siren.ts:33]: No, it's like the brightness for light. I pushed a quick PR in draft mode so we can discuss on it during frontend meeting.

File: src/dialogs/more-info/controls/more-info-siren.ts [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+      return html``;
+    }
+
+    const stateObj = this.stateObj;
+
+    return html`
+      <hr />
+      <div class="actions">
+        ${supportsFeature(stateObj, SirenEntityFeature.TONES) &&
+        this.stateObj.attributes.available_tones
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-siren.ts:63]: If we can only change it when it is on, we should only show it when on right? We do the same for a light and effects.

Comment (piitaya) [reply src/dialogs/more-info/controls/more-info-siren.ts:63]: A use case can be : I want to turn on the siren with this ringtone.

Comment (bramkragten) [reply src/dialogs/more-info/controls/more-info-siren.ts:63]: but it now only does something when it is on already?

**Comment by piitaya** [comment]
I'm closing the PR as the features are designed for automation control, not user control.

**Pull request closed without merging**